.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 coverage
 dist
 docs
+!packages/eslint-plugin-khan/docs
```

Comment:
`docs` was prevent new files from being added in packages/eslint-plugin-khan/docs.